python/_fio_plugin/README.md

```diff
@@ -0,0 +1,44 @@
+# arca-fio
+
+
+
+## Run the thing
+
+### Containerized
+
+#### Requirements
+
+* fio_plugin image named `fio`
```

Comment:
This is confusing. I personally wouldn't name the plugin container as an exact match of the command.